prime-router/src/main/kotlin/azure/FHIRFlowFunctions.kt

```diff
@@ -37,8 +79,10 @@ class FHIRFlowFunctions : Logging {
         val blobContent = content.download()
 
         logger.debug("Got content ${blobContent.size}")
+        // TODO behind an interface?
+        val hl7 = HL7.deserialize(String(blobContent))
```

Comment:
Confused.  What is the difference between the previous function and this part here?

Reply:
This is expanding the ETL part of the fhir work, allow us to test that this desieralization is working as expected for that use case.